styles.css

```diff
@@ -0,0 +1,66 @@
+.header {
+  height: 916px;
+  top: 1px;
+  border-radius: 1px;
+  background-image: url('C:/Users/Justice/Downloads/Microverse-Student-Project-1/Headerbg@2x');
+}
+
```

Comment:
- [ ] Please follow the style guidelines as per Figma template. Kindly stick to the same color, fonts, and background.

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/205117559-0c4987b7-de28-4822-a7df-f5db97676846.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/92215878/205117860-1505d711-25bd-4426-a198-0bb4231dceed.png)

Reply:
As requested by the previous reviewer, kindly stick to the design template as much as you can. I suggest you right click, use inspect and see how your project appears on mobile version. Currently, it does not match the figma design 
- Below screen shot is your design. I have also included an arrow for the button to toggle view between mobile and desktop versions:
<img width="1659" alt="Screenshot 2022-12-02 at 11 10 33" src="https://user-images.githubusercontent.com/48546992/205247196-67e6e3eb-196e-4b83-a2a3-b03edfb861a5.png">

- This is the figma template:
<img width="375" alt="Screenshot 2022-12-02 at 11 12 03" src="https://user-images.githubusercontent.com/48546992/205247248-c74bd6cf-78d3-4ce3-ac1d-c8c2602b0c47.png">